service/service.go

```diff
@@ -93,6 +94,10 @@ func New(config Config) (*Service, error) {
 		}
 	}
 
+	{
+		i, _ := dynamic.NewForConfig(restConfig)
```

Comment:
What is this for? Doesn't seem to be being used and it's causing the e2e test build to fail. Sorry if this is still a WIP. Keeping an eye on these changes as I'm trying to get `aws-operator` tests to work.

Reply:
Yeah this is some leftover.